_sass/_index.scss

```diff
@@ -306,18 +324,35 @@
       font-size: 18px;
       min-height:92px
     }
+    div{
+      padding-left:0 !important;
+      padding-right:5px !important;
+    }
   }
 
-  .mini-panel-text{
+
+  .home_icon_star{
     background-image: url('/images/home_icon_star.svg');
+    @include tablet{
+      background-position: center; 
+    }
+    @include phone{
+      background-position: center; 
+    }
+
   }
-  .mini-panel-call{
+  .home_icon_phone{
     background-image: url('/images/home_icon_map.svg');
```

Comment:
I realize I broke the design a little, but I prefer the left aligned approach even on mobile.  These SVGs don't look good overly tiny, but they become overwhelming when grown in size.